README.md

```diff
@@ -4,4 +4,4 @@ This is the issue tracking repository for Digital Publishing WAI-ARIA Module (dp
 
 You can also compare against [the most recent published version here](https://www.w3.org/TR/dpub-aria-1.0/).
 
-To open a PR, please see the [main ARIA repository where the specification lives](https://github.com/w3c/aria/dpub-aria). This specification is part of the [ARIA suite](https://www.w3.org/WAI/ARIA/deliverables), and uses the same code structure and build tools as other ARIA suites. General information about editing specifications is in the [main ARIA repository readme](https://github.com/w3c/aria/).
+To open a PR, please see the [main ARIA repository where the specification lives](https://github.com/w3c/aria/tree/main/dpub-aria). This specification is part of the [ARIA suite](https://www.w3.org/WAI/ARIA/deliverables), and uses the same code structure and build tools as other ARIA suites. General information about editing specifications is in the [main ARIA repository readme](https://github.com/w3c/aria/).
```

Comment:
```suggestion
To open a PR, please see the [main ARIA repository where the specification lives](https://w3c.github.io/aria/). This specification is part of the [ARIA suite](https://www.w3.org/WAI/ARIA/deliverables), and uses the same code structure and build tools as other ARIA suites. General information about editing specifications is in the [main ARIA repository readme](https://github.com/w3c/aria/).
```

I would think that the goal was to refer to the editors' draft of the document itself. The original reference was indeed a 404, but the proposed change in the PR led to the source in the repository. Hence my proposal for a change of the change...

Reply:
Ah interesting feedback, my intent was to show where the source documents for for the specification resides, thus the link directly to the html document in the ARIA repository.  If someone came to this repository looking for the source documents because they expected them here, I'd guess what they now need is a direct link to the new location of the source documents.

The readme, which you suggested linking to, is also linked in a following sentence "General information about editing specifications is in the [main ARIA repository readme](https://github.com/w3c/aria/)." But perhaps I should say "General information about **contributing to** the specification is in the main ARIA repository readme."

Reply:
sounds good, updated!